src/main/java/com/Alvaeron/api/RPEngineAPI.java

```diff
@@ -45,83 +47,111 @@ public static RoleplayPlayer getRoleplayPlayer(UUID uuid) {
      */
     @SuppressWarnings("deprecation")
     public static RoleplayPlayer getRoleplayPlayer(String playerName) {
-        if (Bukkit.getPlayer(playerName) != null) {
-            return Engine.manager.getPlayer(Bukkit.getPlayer(playerName).getUniqueId());
-        } else if (Bukkit.getOfflinePlayer(playerName) != null) {
-            Player p = (Player) Bukkit.getOfflinePlayer(playerName);
-            if (Engine.manager.getPlayer(p.getUniqueId()) != null) {
-                return Engine.manager.getPlayer(p.getUniqueId());
-            } else {
-                Engine.mm.createRoleplayPlayer(p);
-                return Engine.manager.getPlayer(p.getUniqueId());
-            }
-        } else {
+        Player player = Bukkit.getPlayer(playerName);
+        if (player != null) {
+            return Engine.manager.getPlayer(player.getUniqueId());
+        }
+
+        OfflinePlayer p = Bukkit.getOfflinePlayer(playerName);
+        if (p == null) {
             return null;
         }
+
+        RoleplayPlayer rpPlayer = Engine.manager.getPlayer(p.getUniqueId());
+        if (rpPlayer != null) {
+            return rpPlayer;
+        } else {
+            Engine.mm.createRoleplayPlayer(p);
+            return Engine.manager.getPlayer(p.getUniqueId());
+        }
     }
 
     public static String getRpName(String playerName) {
-        return getRoleplayPlayer(playerName).getName();
+        return Optional.ofNullable(getRoleplayPlayer(playerName))
+            .map(RoleplayPlayer::getName)
+            .orElse("NONE");
     }
 
     public static int getRpAge(String playerName) {
-        return getRoleplayPlayer(playerName).getAge();
+        return Optional.ofNullable(getRoleplayPlayer(playerName))
+            .map(RoleplayPlayer::getAge)
+            .orElse(-1);
```

Comment:
Nitpick: -1 as a default for age seems odd :)

Reply:
DRUE